man/fabtests.7.md

```diff
@@ -15,36 +15,46 @@ Fabtests is a set of examples for fabric providers that demonstrates various fea
 
 # OVERVIEW
   
-Libfabric defines sets of interface that fabric providers can support. The purpose of Fabtests examples is to demonstrate some of the major features. The goal is to familiarize users with different functionalities libfabric offers and how to use them.
+Libfabric defines sets of interface that fabric providers can support. The purpose of Fabtests examples is to demonstrate some of the major features. The goal is to familiarize users with different functionalities libfabric offers and how to use them. Although these tests report performance numbers, they are designed to test functionality and not performance.
 
-The tests are divided into four categories:
+The tests are divided into the following six categories. Except the unit tests all of them are client-server tests.
 
 ## Simple
 
-These tests are a mix of very basic tests and major features of libfabric. All of the tests except info.c are designed to run as client-server processes. A server is started first and then a client process connects to the server and performs various operations.
+These tests are a mix of very basic tests and tests that show major features of libfabric.
 
 	fi_cq_data: A client-server example that tranfers CQ data
 	fi_dgram: A basic DGRAM client-server example
 	fi_dgram_waitset: A basic DGRAM client-server example that uses waitset
 	fi_msg: A basic MSG client-server example
-	fi_msg_pingpong: A ping-pong client-server example using MSG endpoints
-	fi_msg_rma: A ping pong client-server example using RMA operations between MSG endpoints
 	fi_msg_sockets: Verifies transitioning a passive endpoint into an active one as required by sockets-over-RDMA implementations
 	fi_poll: A basic RDM client-server example that uses poll
 	fi_rdm: A basic RDM client-server example
-	fi_rdm_atomic: An RDM ping pong client-server using atomic operations
-	fi_rdm_cntr_pingpong: An RDM ping pong client-server using counters
-	fi_rdm_inject_pingpong: An RDM ping pong client-server example using inject
-	fi_rdm_multi_recv: An RDM ping pong client-server example using multi recv buffer
-	fi_rdm_pingpong: A ping pong client-server example using RDM endpoints
-	fi_rdm_rma: A ping pong client-server example using RMA operations
 	fi_rdm_rma_simple: A simple RDM client-sever RMA example
 	fi_rdm_shared_ctx: An RDM client-server example that uses shared context
-	fi_rdm_tagged_pingpong: A ping pong client-server example using tagged messages
 	fi_rdm_tagged_peek: An RDM client-server example that uses tagged FI_PEEK
 	fi_scalable_ep: An RDM client-server example with scalable endpoints
+
+## Pingpong
+
+The client and the server exchange messages in a ping-pong manner for various messages sizes and measure latencies.
```

Comment:
I would drop the 'and measure latencies' part.  This makes it sound like they are constructed to measure latency, rather than just doing it as a side effect.